includes/post-meta-box.php

```diff
@@ -54,7 +57,29 @@ function meta_box_html( $post ) {
 			</select>
 		</label>
 	</p>
-
+	<p>
+		<label for="podcast_season_number">
+			<?php esc_html_e( 'Season Number', 'simple-podcasting' ); ?>
+			<input type="number" min="0" max="2000" step="1" id="podcast_season_number" name="podcast_season_number" value="<?php echo esc_attr( $season_number ); ?>" />
```

Comment:
What is the purpose of limiting the maximum value for season or episode?

Also, using the number input type doesn't allow to keep it empty and may confuse site owners with the default "0" after the post is saved

Reply:
@cadic During implementation I was thinking that there should be a limit at least without giving that much of a thought. I think it's not necessary and yes the number input might confuse the site owners. Replaced those with text input.